packages/simulation/pitop/simulation/sprites.py

```diff
@@ -240,14 +303,15 @@ def render(self):
 
         image = self.base_image.copy()
         image.blit(
-            self.font.obj.render(str(value), True, pygame.Color("#505050")),
+            self.font.obj.render(str(value), True, PitopColor.CHARCOAL.value),
```

Comment:
probably we need should do pygame.Color in other places like this, though it doesn't seem to be actually broken here